chromium_src/third_party/blink/renderer/core/css/media_values.h

```diff
@@ -6,13 +6,13 @@
 #ifndef BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_CORE_CSS_MEDIA_VALUES_H_
 #define BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_CORE_CSS_MEDIA_VALUES_H_
 
-#define CalculateDeviceWidth                      \
-  CalculateDeviceWidth_ChromiumImpl(LocalFrame*); \
-  static int CalculateDeviceWidth
+#define CalculateDeviceWidth               \
+  CalculateDeviceWidth(LocalFrame*, bool); \
+  static int CalculateDeviceWidth_ChromiumImpl
 
-#define CalculateDeviceHeight                      \
-  CalculateDeviceHeight_ChromiumImpl(LocalFrame*); \
-  static int CalculateDeviceHeight
+#define CalculateDeviceHeight               \
+  CalculateDeviceHeight(LocalFrame*, bool); \
```

Comment:
can this be written as `__VA_ARGS__, bool`? (same for `.cc` file)

Reply:
yes, done